examples/chart/access/datadog/values.yaml

```diff
@@ -0,0 +1,195 @@
+  address: ""
+  # teleport.identitySecretName(string) -- is the name of the Kubernetes secret
+  # that contains the credentials for the connection to your Teleport cluster.
+  #
+  # The secret should be in the following format:
+  #
+  # ```yaml
+  # apiVersion: v1
+  # kind: Secret
+  # type: Opaque
+  # metadata:
+  #   name: teleport-plugin-datadog-identity
+  # data:
+  #   auth_id: #...
+  # ```
+  #
+  # Check out the [Access Requests with
+  # Datadog Incident Management](../../admin-guides/access-controls/access-request-plugins/datadog-hosted.mdx) guide
+  # for more information about how to acquire these credentials.
+  identitySecretName: ""
+  # teleport.identitySecretPath(string) -- is the key in the Kubernetes secret
+  # specified by `teleport.identitySecretName` that holds the credentials for
+  # the connection to your Teleport cluster. If the secret has the path,
+  # `"auth_id"`, you can omit this field.
+  identitySecretPath: "auth_id"
+
+# datadog -- contains the configuration used by the plugin to authenticate to Datadog.
+#
+# You can pass the Datadog keys by setting the chart values or using an existing Kubernetes Secret.
+datadog:
+  # datadog.apiEndpoint(string) -- specifies which Datadog API site to set API
+  # requests.
+  apiEndpoint: "https://api.datadoghq.com"
+  # datadog.apiKey(string) -- is the Datadog API key used by the plugin to interact
+  # with Datadog. When set, the Chart creates a Kubernetes Secret for you.
+  #
+  # This value has no effect if `datadog.apiKeyFromSecret` is set.
+  apiKey: ""
+  # datadog.apiKeyFromSecret(string) -- is the name of the Kubernetes Secret
+  # containing the Datadog apiKey. When this value is set, you must create the
+  # Secret before creating the chart release.
+  apiKeyFromSecret: ""
+  # datadog.apiKeySecretPath(string) -- is the Kubernetes Secret key
+  # containing the Datadog API key. The secret name is set via `datadog.apiKeyFromSecret`.
+  apiKeySecretPath: "datadogApiKey"
+  # datadog.applicationKey(string) -- is the Datadog Application key used by the plugin to interact
+  # with Datadog. When set, the Chart creates a Kubernetes Secret for you.
+  #
+  # This value has no effect if `datadog.applicationKeyFromSecret` is set.
+  applicationKey: ""
+  # datadog.applicationKeyFromSecret(string) -- is the name of the Kubernetes Secret
+  # containing the Datadog applicationKey. When this value is set, you must create the
+  # Secret before creating the chart release.
+  applicationKeyFromSecret: ""
+  # datadog.applicationKeySecretPath(string) -- is the Kubernetes Secret key
+  # containing the Datadog Application key. The secret name is set via `datadog.applicationKeyFromSecret`.
+  applicationKeySecretPath: "datadogApplicationKey"
+  # datadog.fallbackRecipient(string) -- specifies the default recipient for
+  # Access Request notifications. The recipient can be a Datadog user email or
+  # a team handle.
+  fallbackRecipient: "admin@example.com"
```

Comment:
Can you remove the default value and put the example email in the comment/godoc? Users will ignore this field and the plugin will try notify non-existing users.